schemas/com.mparticle.snowplow/appstatetransition_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,40 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an app state transition event",
+	"self": {
+		"vendor": "com.mparticle.snowplow",
+		"name": "appstatetransition_event",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"transitionType": {
+			"type": "string"
```

Comment:
No maxLengths on any strings?

Reply:
Is this required or just preferred? I'd prefer to keep string types as flexible (or "future-proof") as possible.

Reply:
It's not required, but in Redshift you will end up with the Igluctl default lengths for every string - so you are not gaining a lot of flexibility...